### DISCUSSION TITLE
Benutzerdefinierte Autotype-Sequenz benutzen

### DISCUSSION THREAD

**Discussion by c-atmbu** [keepassxreboot/keepassxc#9681] [Q&A]
Ich würde gerne eine benutzerdefinierte Autotype-Sequenz erstellen und habe jetzt schon alle möglichen Versionen versucht. Allerdings ohne Erfolg. Ich füge einen Screen-Shot meiner Bemühungen bei. Was mache ich verkehrt? Es funktioniert einfach nicht. Vielen Dank für die Unterstützung. 

![Bildschirmfoto vom 2023-07-31 20-54-04](https://github.com/keepassxreboot/keepassxc/assets/69669282/86165985-b62d-415d-a54b-a56acabb5ea3)

**Comment by droidmonkey** [comment]
Did you put the actual attribute name in the `{S: ...}` part?

**Comment by c-atmbu** [comment]
Kann ich denn den Attributenamen individuell vergeben und wo lege ich ihn fest. 
![Bildschirmfoto vom 2023-08-01 08-39-39](https://github.com/keepassxreboot/keepassxc/assets/69669282/80e0a771-cd1a-4697-8d4a-e7f96e199b4a)
